src/com/lftechnology/java/training/sanish/numbergame/GuessNumberGame.java

```diff
@@ -0,0 +1,137 @@
+package com.lftechnology.java.training.sanish.numbergame;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Guess number program in between 1 - 20
+ * 
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class GuessNumberGame {
+	private static final Logger LOGGER = Logger
+			.getLogger(GuessNumberGame.class.getName());
+	private static final int MIN_RANDOM_NUM = 1;
+	private static final int MAX_RANDOM_NUM = 20;
+
+	private int randomNum;
+	private int guessNum;
+	private int guessCount = 0;
+
+	/**
+	 * Constructor
+	 * 
+	 * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+	 */
+	public GuessNumberGame() {
+		int randNum = (int) (Math.random() * MAX_RANDOM_NUM) + MIN_RANDOM_NUM;
+		setRandomNum(randNum);
+	}
+
+	/**
+	 * Return randomNum value
+	 * 
+	 * @return {@link Integer}
+	 */
+	public int getRandomNum() {
+		return randomNum;
+	}
+
+	/**
+	 * Set randomNum value
+	 * 
+	 * @param randomNum
+	 *            {@link Integer}
+	 */
+	public void setRandomNum(int randomNum) {
+		this.randomNum = randomNum;
+	}
+
+	/**
+	 * Return guessNum value
+	 * 
+	 * @return {@link Integer}
+	 */
+	public int getGuessNum() {
+		return guessNum;
+	}
+
+	/**
+	 * Set guessNum value. If parameter guessNum value greater than max random
+	 * number set value then throw exception
+	 * 
+	 * @param guessNum
+	 *            {@link Integer} user guess number
+	 * @throws Exception
+	 *             Guess number should be less or equal to [MAX_RANDOM_NUM]
+	 * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+	 */
+	public void setGuessNum(int guessNum) throws Exception {
+		if (guessNum >= MIN_RANDOM_NUM && guessNum <= MAX_RANDOM_NUM) {
+			this.guessNum = guessNum;
+		} else {
+			throw new Exception("Guess number should be in range form "
+					+ MIN_RANDOM_NUM + " to " + MAX_RANDOM_NUM);
+		}
+	}
+
+	/**
+	 * Return guessCount value
+	 * 
+	 * @return {@link Integer}
+	 */
+	public int getGuessCount() {
+		return guessCount;
+	}
+
+	/**
+	 * Set guessCount value
+	 * 
+	 * @param guessCount
+	 *            {@link Integer}
+	 */
+	public void setGuessCount(int guessCount) {
+		this.guessCount = guessCount;
+	}
```

Comment:
Ok, And does I required add documentation if I modified setter? Here I have modified setGuessNum function, so I added documentation and then I added to all other getter and setter functions for uniformity.